setup.py

```diff
@@ -81,7 +84,7 @@
 extras_require['all'] = extras_require['proxy']
```

Comment:
```suggestion
```

Not the subject of this PR, but with everything else, it just doesn't feel like `all` does what people will expect it to do. Not sure if there are other references.